app/services/designation_and_undesignation/base.rb

```diff
@@ -0,0 +1,62 @@
+module DesignationAndUndesignation
+  class Base
+    attr_accessor :order_id, :package, :quantity, :is_new_orders_package
+
+    def designate_stockit_item
+      package.designate_to_stockit_order(order_id)
+    end
+
+    def designated_orders_packages
+      package.get_designated_orders_packages
+    end
+
+    def dispatched_location_id
+      id ||= Location.dispatch_location.id
```

Comment:
Are you using this to memoize the result? If so, use @id otherwise `id` is just a local variable and it's value is lost when the method returns.